src/main/resources/hudson/plugins/sshslaves/verifiers/ManuallyProvidedKeyVerificationStrategy/config.jelly

```diff
@@ -35,7 +35,7 @@ THE SOFTWARE.
     </j:if>           
 
 	<f:entry title="${%SSH Key}" field="key">
-		<f:textarea />
+		<f:textarea checkMethod="post"/>
```

Comment:
This isn't leaking information about the key, is it? I don't see any permission checks.

Reply:
It is a public key so it would not be a security issue, however, I'd open an issue to check every do* to use POST and require proper permission to avoid abuse and security issues.

Reply:
https://issues.jenkins-ci.org/browse/JENKINS-58691